apps/website/src/pages/docs/expandableblock.mdx

```diff
@@ -9,67 +9,83 @@ group: core
+The borderless expandable block is useful in tighter areas where using a default expandable block would visibly cause too many blocks. The borderless expandable block is achieved by using the `styleType` prop.
 
-The visual appearance of an expandable block is different from an expandable row. An expandable block can not be selected / checked like an expandable row can, therefore it is visually styled differently.
+<LiveExample src='ExpandableBlock.borderless.tsx'>
+  <AllExamples.ExpandableBlockBorderlessExample client:load />
+</LiveExample>
 
 ## Usage
 
-### Expandable rows
+- Expandable blocks are used to progressively disclose information. [Read this article to learn more about progressive disclosure.](https://www.nngroup.com/articles/progressive-disclosure/)
 
-Expandable rows are used in tables. The layout inside expanded rows varies according to requirements. Child rows can be selectable and be an entry point to more content (ex.: downloadable files), or only contain metadata about the parent file. If at least one row within a table is expandable, all rows should follow the same pattern. Avoid mixing regular table rows with expandable rows.
+- Expandable blocks should not be used with flattened hierarchy or information panels.
 
-### Expandable blocks
+### Forms
 
 Expandable blocks are used in pages to divide up content in sections, in forms for example. If each section has many fields, using expandable blocks will help alleviate the UI and prevent the user from scrolling endlessly to get to the section they are looking for.
 
-Expanded block containing form elements within an interface.
+<LiveExample src='ExpandableBlock.form.tsx'>
+  <AllExamples.ExpandableBlockFormExample client:load />
+</LiveExample>
 
 ### Expandable blocks VS hierarchy tree
 
-While these two patterns may appear similar, they both fulfill a different role and are not interchangeable. Here are some tips to determine if an expandable block or row is what you need.
+While these two patterns may appear similar, they both fulfill a different role and are not interchangeable. Here are some tips to determine if an expandable block is what you need.
 
-Use an expandable rows/blocks when:
+Use an expandable block when:
 
-- You need a row in a table to display additional metadata that doesn’t align with the column content;
 - You have a page with different sections all containing user-generated content that would benefit from being split up;
 - There is only one level of indentation possible at all times;
 - The content you want to be expandable varies from simple list items to more complex form components (such as text fields, buttons, etc.).
 
-Use a hierarchy tree when:
+Use a [hierarchy tree](tree) when:
 
 - You require a pattern that works with several levels of data (folder drilling);
 - There are only list items in the pattern;
```

Comment:
Could you remove the semicolons and periods at the end of these bullet points? It's kind of an odd thing to do with bullet points.

Reply:
Fixed

Reply:
This has been updated